src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -89,32 +92,72 @@ private void RenderCurrentViewAndCompare(string testName)
             var refbitmap = BitmapFromSource(refimage);
             var newImage = BitmapFromSource(bitmapsource);
 
+#if SAVEDEBUGIMAGES
```

Comment:
What is this for?

Reply:
this will dump the new renders into the same folder where the reference images are so you can easily compare them and see what the current render looks like.